demo/combo-box-theme-variants-demos.html

```diff
@@ -0,0 +1,47 @@
+<dom-module id="combo-box-theme-variants-demos">
+  <template>
+    <style include="vaadin-component-demo-shared-styles">
+      :host {
+        display: block;
+      }
+    </style>
+    <h3>Text align</h3>
+    <vaadin-demo-snippet id="theme-variants" when-defined="vaadin-combo-box">
```

Comment:
The other theme variant demo has id "small-size", so should this then be "text-align" rather than general "theme-variants"?

Reply:
Done